OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -325,5 +332,44 @@ class StatsDropDownOption
 			public Func<bool> IsSelected;
 			public Action OnClick;
 		}
+
+		public bool HandleKeyPress(KeyInput e)
+		{
+			if (e.Event == KeyInputEvent.Down && !e.IsRepeat)
+			{
+				var h = Hotkey.FromKeyInput(e);
+				if (h == Game.Settings.Keys.StatisticsBasicKey)
```

Comment:
sorry, I pushed only rebased stuff, forgot to stage before commit. I don't use Keys.StatisticsBasicKey,... anymore.